src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/ConfigureApplicationPartManager.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Mvc.ApplicationParts
+{
+    public abstract class ConfigureApplicationPartManager
```

Comment:
I'm not sure I understand this class. Why does configure take both the part manager and a part model?

Reply:
My plan was to have a couple of default implementations for Configure in the Mvc - `Null` and `ConfigureCompiledView`, the latter would be a property we stamp on Sdk generated views. The action needs to know what to configure and what to configure it with.